src/pages/_app.jsx

```diff
@@ -4,12 +4,17 @@ import 'bootstrap/dist/css/bootstrap.css';
 
 const Application = ({Component, pageProps}) => (
   <>
-    <nav className='navbar navbar-dark bg-primary'>
+    <nav className='navbar navbar-expand-lg sticky-top navbar-dark bg-primary'>
       <div className='container'>
         <Link href='/'>
           <a className='navbar-brand'>Observatorio</a>
         </Link>
-      </div>
+        <a class="nav-item nav-link text-light" href="./report">Informes</a>
+        <a class="nav-item nav-link text-light" href="./campaign">Campañas</a>
+        <a class="nav-item nav-link text-light" href="./mediapresence">En Los Medios</a>
+        <a class="nav-item nav-link text-light" href="./activity">Actividades</a>
+        <a class="nav-item nav-link text-light" href="./about">Quiénes Somos</a>
```

Comment:
Two comments on these lines:
* `class` should be `className` here, or else it won't be pick up by react. See [the docs on jsx attributes](https://reactjs.org/docs/introducing-jsx.html#specifying-attributes-with-jsx)
* You shouldn't be using `href` directly here. Using `href` directly messes up with client-side navigation (the link goes to the server instead of doing a client-side page change). Use [next/Link](https://nextjs.org/docs/api-reference/next/link) instead, like what we are doing  on the `navbar-brand` item above.